CHANGELOG.md

```diff
@@ -10,6 +10,12 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 
 ### Changed
 
+## [v2.0.1] 2021-2-5
+
+### Changed
+* *ORCA-125* BucketOwnerFullControl ACL is now set on PUT requests to buckets.
```

Comment:
I would add to the information here about the lambda it effects. Maybe something like the following

*ORCA-125* BucketOwnerFullControl ACL is now set on for storage PUT requests in the copy_to_glacier lambda. This prevents errors during cross account (OU) copying of data.

Reply:
Fixed.